core/src/filed/fd_plugins.cc

```diff
@@ -848,17 +853,20 @@ int PluginSave(JobControlRecord* jcr, FindFilesPacket* ff_pkt, bool)
 
       if (retval == bRC_More) { continue; }
```

Comment:
i cannot make a note on the right line, but a few lines above this, you need to write
```
bRC retval = PlugFunc(...);
```